docs/UserCommands.md

```diff
@@ -0,0 +1,44 @@
+## User Command Reference
+
+
+`@rocket user`
+
+## Options to specify input:
+
+* `-edit`
+    * `-name` somename `-email` email `-pos` yourposition `-github` githublink `-major` yourmajor `-bio` yourbio
+        * edit properties of your Launch Pad profile
+    * ADMIN only option: `-member` MEMBER_ID
+        * edit properties of another user's profile
+
+
+* `-view` MEMBER_ID
+    * view information about a user
+
+
+* `-add`
+     * `-team` TEAM_ID `-member` MEMBER_ID
+        * add user to team
+
+
+* `-rmv`
+    * `-team` TEAM_ID `-member` MEMBER_ID
+        * remove user from team
+
+
+* `-help`
+    * outputs options for user commands
+
+
+## ADMIN only:
+
+* `-delete` MEMBER_ID
+    * permanently delete member's Launch Pad profile
+
+
+* `-add_admin` MEMBER_ID
```

Comment:
We should also have commands to create/remove team leads.

Reply:
Should that be in user or in team commands?

Reply:
The "team lead" permission level is going to be independent of actually leading a team (come to think of that, that's kind of confusing... maybe we should rename it in `model/permissions.py`?) and I want the commands to change permissions levels to be consistent, so `add_admin` and `add_team_lead` should be in the same place.